### PR TITLE
[device_info] Changed directory of device_info

### PR DESCRIPTION
**Pull request by yash1200**
## Description

In this pr, I have moved everything from device_info to device_info/device_info.

## Checklist

Before you create this PR confirm that it meets all requirements listed below by checking the relevant checkboxes (`[x]`). This will ensure a smooth and quick review process.

- [x] I read the [Contributor Guide] and followed the process outlined there for submitting PRs.
- [x] My PR includes unit or integration tests for *all* changed/updated/fixed behaviors (See [Contributor Guide]).
- [x] All existing and new tests are passing.
- [x] I updated/added relevant documentation (doc comments with `///`).
- [x] The analyzer (`flutter analyze`) does not report any problems on my PR.
- [x] I read and followed the [Flutter Style Guide].
- [x] The title of the PR starts with the name of the plugin surrounded by square brackets, e.g. [shared_preferences]
- [x] I updated pubspec.yaml with an appropriate new version according to the [pub versioning philosophy].
- [x] I updated CHANGELOG.md to add a description of the change.
- [x] I signed the [CLA].
- [x] I am willing to follow-up on review comments in a timely manner.

## Breaking Change

Does your PR require plugin users to manually update their apps to accommodate your change?

- [ ] Yes, this is a breaking change (please indicate a breaking change in CHANGELOG.md and increment major revision).
- [x] No, this is *not* a breaking change.

[issue database]: https://github.com/flutter/flutter/issues
[Contributor Guide]: https://github.com/flutter/plugins/blob/master/CONTRIBUTING.md
[Flutter Style Guide]: https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo
[pub versioning philosophy]: https://www.dartlang.org/tools/pub/versioning
[CLA]: https://cla.developers.google.com/

### CODE REVIEW

File: packages/device_info/device_info/example/pubspec.yaml [hunk 1]
```diff
@@ -4,14 +4,13 @@ description: Demonstrates how to use the device_info plugin.
 dependencies:
   flutter:
     sdk: flutter
+  e2e:
```
Comment (ditman) [inline packages/device_info/device_info/example/pubspec.yaml:7]: Why move this to a normal dependency? Please leave it as a dev_dependency, and fix the path: `../../../e2e`?

Comment (yash1200) [reply packages/device_info/device_info/example/pubspec.yaml:7]: It was giving tests error when I made a PR last time. I'll reverse the change right now.

Comment (ditman) [reply packages/device_info/device_info/example/pubspec.yaml:7]: Thanks! I think I've seen the errors that you're describing, and they mean that you need to add e2e as a dev_dependency on the core plugin, not only on the example (I think it needs to be on both :P)

Let's confirm with the CI!

Comment (yash1200) [reply packages/device_info/device_info/example/pubspec.yaml]: I took the example from the url_launcher plugin and it didn't had e2e in the core plugin's pubspec.yaml. e2e was only present in the example. Lets confirm with the CI.

**Comment by ditman** [comment]
(The `build_all_plugins_ipa` was fixed last night. Please rebase your branch with the latest master to get that test to pass!)

**Comment by yash1200** [comment]
> (The `build_all_plugins_ipa` was fixed last night. Please rebase your branch with the latest master to get that test to pass!)

Sure sir!. I'm on it.

File: packages/google_maps_flutter/google_maps_flutter/CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,12 @@
+## 0.5.30
```
Comment (ditman) [inline packages/google_maps_flutter/google_maps_flutter/CHANGELOG.md:1]: Odd that you got changes from google_maps_flutter, what command did you use to merge/rebase your branch?

Comment (yash1200) [reply packages/google_maps_flutter/google_maps_flutter/CHANGELOG.md:1]: First i merge changes to master branch using upstream and then made to merge it in my current branch , I used git rebase master and git merge origin/yash_device_info and then commited and 
pushed the changes

Comment (ditman) [reply packages/google_maps_flutter/google_maps_flutter/CHANGELOG.md:1]: Hmm, odd, normally you either rebase or merge after updating your origin/master (that's how I do it! :P)

We'll see when we merge this, maybe you become the forever owner of methods in other plugins ;)

Comment (ditman) [reply packages/google_maps_flutter/google_maps_flutter/CHANGELOG.md:1]: (Don't worry too much, I think github is going to be able to manage :P)

**Comment by ditman** [comment]
I'll merge this once the tests in master finish. Thank you very much for your contribution, @yash1200! (And your patience :P)

File: packages/google_maps_flutter/google_maps_flutter/CHANGELOG.md (hunk 2 above)
Comment (yash1200) [reply packages/google_maps_flutter/google_maps_flutter/CHANGELOG.md:1]: Yes I did updated my origin/master first and then merged the changes in the local branch. I don't know what went wrong. Let me know if i can do anything to fix this.

**Comment by yash1200** [comment]
@ditman Thank you sir. Really means a lot.

**Pull request merged**